Tests/NIOHPACKTests/HPACKCodingTests.swift

```diff
@@ -500,25 +500,15 @@ class HPACKCodingTests: XCTestCase {
             XCTAssertEqual(err.allowedSize, 4096)
         }
         
-        do {
-            _ = try decoder.decodeHeaders(from: &request3)
-            XCTFail("Decode should have failed with InvalidDynamicTableSize")
-        } catch _ as NIOHPACKErrors.InvalidDynamicTableSize {
-            // this is expected
-        } catch {
-            XCTFail("Unexpected error: \(error)")
+        XCTAssertThrowsError(try decoder.decodeHeaders(from: &request3)) {error in
+            XCTAssertNotNil(error as? NIOHPACKErrors.InvalidDynamicTableSize)
```

Comment:
This would be better written as `XCTAssertTrue(error is NIOHPACKErrors.InvalidDynamicTableSize)`. The same applies to anywhere else you're using `XCTAssertNotNil` and an `as?`.

Reply:
Thanks for the review. I have done the change in new commit.